ENDOC-710-tomcat-default edits (#746)
entando/entando-docs: vuepress/docs/next/docs/reference/deployment-structure.md

The **entando-de-app** is a J2EE application and an instance of the [entando-core](#entando-core). It provides pathways for Entando Components and the server image required by the Entando Operator to manage the deployment. The pom.xml for the application reveals its dependencies.

> * GitHub: <https://github.com/entando/entando-de-app/>

Reviewer: Pretty sure this should be changed to this - https://github.com/entando/app-engine. CC: @pietrangelo @eugeniosant

> * DockerHub: 

Author: @pietrangelo need the new DockerHub addr for the 7.2 de-app

Reviewer: https://registry.hub.docker.com/r/entando/entando-de-app-eap, https://registry.hub.docker.com/r/entando/entando-de-app-wildfly, https://registry.hub.docker.com/r/entando/entando-de-app-tomcat

#### Customization

> The **entando-de-app** is very likely to be customized as part of an Entando implementation. 

Reviewer: We're trying to discourage customization by providing better extension points.
```suggestion
The **entando-de-app** is sometimes customized as part of an Entando implementation. 
```
